.travis.yml

```diff
@@ -0,0 +1,42 @@
+dist: trusty
+language: go
+go:
+  - 1.11
```

Comment:
I'm not positive but this syntax might cause it to lock to `1.11.0`. I've been using the following syntax
```
  - "1.11.x"
```

Reply:
Okie dokie - works for me!